csrc/scheduler/normalization_utils.h

```diff
@@ -191,5 +191,13 @@ int64_t getInnerOuterPersistentBufferBatches(
     const int64_t outer_dim_numel,
     const int64_t warpSize);
 
+//! Each thread can use a maximum of 255 registers, and assume 40 of them are
+//! reserved for indexing and other purposes. So, each thread can use up to 215
+//! registers for persistent buffer. Calculate number of buffer batches using
+//! these 215 registers.
+int64_t getMaximumInnerOuterPersistentBufferBatch(
+    const int64_t total_buffer_bytes,
+    const int64_t reduction_elements,
+    const int64_t vectorization_factor);
```

Comment:
Can you please comment about these parameters?

Reply:
added